### ISSUE TITLE
log scaling on scatter pushes negative numbers out of the visualization

### ISSUE THREAD

**Issue by metasyn** [alexandersimoes/d3plus#335]
![screen shot 2015-06-12 at 4 31 26 pm](https://cloud.githubusercontent.com/assets/1719730/8141745/83181fa0-1120-11e5-9ff4-fdce220556d9.png)

**Comment by curran** [comment]
Log scale by its nature only works on positive values. It has to take the log of the input value, and `log(x)` is undefined for negative `x`. I'm curious, what behavior would you expect?

**Comment by metasyn** [comment]
Yeah, I looked further and I think there were some zeros in the data. I guess, the current behavior doesn't looks intentional in the sense that the points are half displayed and off the axis.

**Comment by davelandry** [comment]
Presumed fixed in 2.0
